install_poetry/install_poetry.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#  Copyright (c) 2022 Ecco Sneaks & Data
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -Eeuo pipefail
+
+curl -sSL https://install.python-poetry.org | python3 - --preview
+export PATH=/github/home/.local/bin:$PATH
+poetry config repositories.azops "$REPO_URL"
+poetry install
```

Comment:
`poetry install` is not always the case. For example, in Proteus build we do `poetry install --extras ...`, check here
https://github.com/SneaksAndData/proteus/blob/main/.github/workflows/build.yaml#L29

Reply:
Added parameters to control installation of extras and additional poetry options.